pyproject.toml

```diff
@@ -7,8 +7,17 @@ build-backend = "hatchling.build"
 name = "zarr"
 description = "An implementation of chunked, compressed, N-dimensional arrays for Python"
 readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "Alistair Miles", email = "alimanfoo@googlemail.com" },
+]
 maintainers = [
-    { name = "Alistair Miles", email = "alimanfoo@googlemail.com" }
+    { name = "Josh Moore", email = "josh@openmicroscopy.org" },
+    { name = "Juan Nunez-Iglesias", email = "juan.nunez-iglesias@monash.edu" },
+    { name = "Joe Hamman", email = "joe@earthmover.io" },
+    { name = "Davis Bennett", email = "davis.v.bennett@gmail.com" },
+    { name = "jakirkham"},
+    { name = "Martin Durant", email = "mdurant@anaconda.com" },
+    { name = "Norman Rzepka"},
```

Comment:
@zarr-developers/python-core-devs - I've added your name here. If you had your email listed on your github profile, I added it to. Let me know if you'd like to change this.